server/lib/template/partials/title.handlebars

```diff
@@ -1,6 +1,6 @@
 <!-- this partial requires binding the variable 'metadata' -->
 
-<div class="col-xs-7 col-auto">
+<div class="col-auto">
```

Comment:
Hmm... we use `col-auto` several times in BHIMA, but I cannot figure out where it is defined.  I don't think we actually have a definition for it.  Maybe we just remove it?

![image](https://github.com/user-attachments/assets/d3ad21ca-e345-4ba9-af4d-b60a40f1b1e1)

Reply:
Done